docs/typescript/tutorials/logging.md

```diff
@@ -43,16 +35,18 @@ You can access your current span via [`DBOS.span`](../reference/transactapi/dbos
 
 ### OpenTelemetry Export
 
-You can export DBOS logs and traces to any OpenTelemetry Protocol (OTLP)-compliant receiver.
-In DBOS Cloud, this is done automatically, and you can view your logs and traces in the [cloud console](https://console.dbos.dev/login-redirect).
+You can export DBOS traces to any OpenTelemetry Protocol (OTLP)-compliant receiver.
 
-Locally, you can configure exporters in your [`dbos-config.yaml`](../reference/configuration.md):
+You can [configure](../reference/configuration.md) a custom export target.
+For example:
 
-```yaml
-telemetry:
-    OTLPExporter:
-        logsEndpoint: http://localhost:4318/v1/logs
-        tracesEndpoint: http://localhost:4318/v1/traces
+```javascript
+DBOS.setConfig({
+  name: 'my-app',
+  databaseUrl: process.env.DBOS_DATABASE_URL,
+  otlpTracesEndpoints: ["http://localhost:4318/v1/traces"],
```

Comment:
Note Transact TS supports only 1 exporter -- we'll add support for more soon!

Reply:
Regardless, the interface requires a list